libunwind/include/libunwind.h

```diff
@@ -560,6 +560,40 @@ enum {
   UNW_ARM64_D29 = 93,
   UNW_ARM64_D30 = 94,
   UNW_ARM64_D31 = 95,
+#ifdef __CHERI_PURE_CAPABILITY__
+  UNW_ARM64_C0 = 198,
+  UNW_ARM64_C1 = 199,
+  UNW_ARM64_C2 = 200,
+  UNW_ARM64_C3 = 201,
+  UNW_ARM64_C4 = 202,
+  UNW_ARM64_C5 = 203,
+  UNW_ARM64_C6 = 204,
+  UNW_ARM64_C7 = 205,
+  UNW_ARM64_C8 = 206,
+  UNW_ARM64_C9 = 207,
+  UNW_ARM64_C10 = 208,
+  UNW_ARM64_C11 = 209,
+  UNW_ARM64_C12 = 210,
+  UNW_ARM64_C13 = 211,
+  UNW_ARM64_C14 = 212,
+  UNW_ARM64_C15 = 213,
+  UNW_ARM64_C16 = 214,
+  UNW_ARM64_C17 = 215,
+  UNW_ARM64_C18 = 216,
+  UNW_ARM64_C19 = 217,
+  UNW_ARM64_C20 = 218,
+  UNW_ARM64_C21 = 219,
+  UNW_ARM64_C22 = 220,
+  UNW_ARM64_C23 = 221,
+  UNW_ARM64_C24 = 222,
+  UNW_ARM64_C25 = 223,
+  UNW_ARM64_C26 = 224,
+  UNW_ARM64_C27 = 225,
+  UNW_ARM64_C28 = 226,
+  UNW_ARM64_C29 = 227,
+  UNW_ARM64_C30 = 228,
+  UNW_ARM64_C31 = 229,
```

Comment:
```suggestion
  UNW_ARM64_C31 = 229,
  UNW_ARM64_CSP = 229,
```
to match UNW_ARM64_SP